short-read-mngs/idseq_utils/idseq_utils/batch_run_helpers.py

```diff
@@ -84,46 +113,128 @@ def _run_batch_job(job_name, job_queue, job_definition, environment, chunk_id, a
         except ClientError as e:
             # If we get throttled, randomly wait to de-synchronize the requests
             if e.response["Error"]["Code"] == "TooManyRequestsException":
-                log.warn("describe_jobs_rate_limit_error", extra={"job_id": job_id})
+                log.warn(f"describe_jobs_rate_limit_error for job_id: {job_id}")
                 # Possibly implement a backoff here if throttling becomes an issue
             else:
                 log.error(
-                    "unexpected_client_error_while_polling_job_status",
-                    extra={"job_id": job_id},
+                    f"unexpected_client_error_while_polling_job_status for job_id: {job_id}",
                 )
                 raise e
 
         if status == "SUCCEEDED":
-            _log_alignment_batch_job_status(
-                job_id, job_queue, job_definition, chunk_id, status, alignment_algorithm
-            )
+            _log_status(status)
             return job_id
         if status == "FAILED":
-            log.error(
-                "alignment_batch_job_failed",
-                extra={
-                    "job_id": job_id,
-                    "chunk_id": chunk_id,
-                    "alignment_algorithm": alignment_algorithm,
-                },
-            )
-            _log_alignment_batch_job_status(
-                job_id, job_queue, job_definition, chunk_id, status, alignment_algorithm
-            )
+            _log_status(status)
             raise BatchJobFailed("chunk alignment failed")
         time.sleep(delay)
 
 
+def _run_chunk(
+    input_dir: str,
+    chunk_dir: str,
+    aligner: str,
+    aligner_args: str,
+    queries: List[str],
+    chunk_id: int,
+    db_chunk: str,
+):
+    deployment_environment = os.environ["DEPLOYMENT_ENVIRONMENT"]
+    pattern = r"s3://.+/samples/([0-9]+)/([0-9]+)/"
+    m = re.match(pattern, input_dir)
+    if m:
+        project_id, sample_id = m.group(1), m.group(2)
+    else:
+        project_id, sample_id = "0", "0"
```

Comment:
it actually might be better to fail here to avoid confusion

Reply:
That would preclude us from running ad-hoc workflows not from the application.

Reply:
Ah good point